booking-app/components/src/client/routes/components/Provider.tsx

```diff
@@ -175,8 +180,10 @@ export const DatabaseProvider = ({
     const liaisonEmails = liaisonUsers.map((liaison) => liaison.email);
     const paEmails = paUsers.map((pa) => pa.email);
     const equipmentEmails = equipmentUsers.map((e) => e.email);
+    const superAdminEmails = superAdminUsers.map((admin) => admin.email);
 
     // Check permissions
+    if (superAdminEmails.includes(userEmail)) return PagePermission.SUPER_ADMIN;
```

Comment:
@pearmini 
Instead of growing the single-value PagePermission enum and adding more || checks, we could store permissions as tags in an array – users could hold multiple roles, cross-tenant (super_admin) stays orthogonal, and page guards become one helper call.

```
// permissions.ts
export type Permissions = 'booking' | 'pa' | 'liaison' | 'equipment' | 'admin' | 'super_admin';
export const hasPermissions = (u: Permission[], req: Permission[]) => req.some(p => u.includes(p));

// usage in Provider
if (hasPerm(userPerms, ['admin', 'super_admin'])) { … }
```